scripts/preprocess/process_compute_centroids.R

```diff
@@ -0,0 +1,22 @@
+process_compute_centroids <- function(outfile, spdat, county_dict_file) {
+  
+  # compute the centroids (actually, return a point on the surface of each multipolygon)
+  centroids <- rgeos::gPointOnSurface(spdat, byid=TRUE, id=spdat@data$GEOID)
```

Comment:
can you just add a note about how this important for groups of islands compared to `gCentroid`? I want to remember that for the future, but I have a terrible memory and prefer things to be written :)

Reply:
yes, coming